.evergreen/.evg.yml

```diff
@@ -2151,3 +2151,21 @@ buildvariants:
   tasks:
     - name: testazurekms_task_group
       batchtime: 20160 # Use a batchtime of 14 days as suggested by the CSFLE test README
+
+- name: container-variant
```

Comment:
Where can we look in Evergreen to observe the effect this had?

Reply:
Bump

Reply:
@jeff [Here](https://spruce.mongodb.com/version/64db97dd32f4175a15c1e989/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) is a sample patch run in Evergreen showing tasks under a container-based variant. Selecting `Reconfigure tasks/variants` will show all build variants that exist as part of this change